app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -39,8 +39,9 @@
   @include govuk-clearfix;
   background-color: govuk-colour("blue");
   color: govuk-colour("white");
-  padding: govuk-spacing(2) govuk-spacing(3) 0;
+  padding: govuk-spacing(5);
   outline: 0;
+  border: solid 1px govuk-colour("white");
```

Comment:
Putting a border all the way around the element will make it appear 2px narrower than the page width it's supposed to fit into. Suggest border bottom only.

Reply:
have adjusted